Views/backend/connect/view/export/product/progress.js

```diff
@@ -89,12 +100,15 @@ Ext.define('Shopware.apps.Connect.view.export.product.Progress', {
                 }
                 me.startButton.setDisabled(true);
                 me.cancelButton.setDisabled(true);
-
-                me.fireEvent('startExport', me.sourceIds, me.batchSize, me);
+                if (me.exportAll) {
```

Comment:
i think the progress bar shouldn't fire any event at all
it should just show some progress that will be updated but shouldn't know anything more
i would suggest firing this event after creating this progess bar